rhscl-charter.md

```diff
@@ -209,6 +209,8 @@ Daemon is listening on 0.0.0.0 by default.
 
 Data directory (if any) that is expected to be mounted shouldn't be home directory of user, there may be more stuff that we don't want to mount, so in some cases we use data/ subdirectory for data themselves (and VOLUME)
 
+Daemon log errors to `stdout` or `stderr`, so the logs are shown in the `docker log` output. This will probably change as soon as we can use `systemd` in the container and proper `journald` logging.
+
```

Comment:
It will not be solved by systemd running inside the container (maybe yes if the systemd inside container can forward logs to the host systemd)...
We will use something like this for logging: http://www.projectatomic.io/blog/2015/04/logging-docker-container-output-to-journald but it will still require that daemons have to log to stdout/stderr. 

@rhatdan do you know ETA of journald log driver lands in Docker?